xrpl/transaction/tx_type.go

```diff
@@ -3,24 +3,31 @@ package transaction
 type TxType string
 
 const (
-	AccountSetTx                        TxType = "AccountSet"
-	AccountDeleteTx                     TxType = "AccountDelete"
-	AMMBidTx                            TxType = "AMMBid"
-	AMMCreateTx                         TxType = "AMMCreate"
-	AMMDeleteTx                         TxType = "AMMDelete"
-	AMMDepositTx                        TxType = "AMMDeposit"
-	AMMVoteTx                           TxType = "AMMVote"
-	AMMWithdrawTx                       TxType = "AMMWithdraw"
-	CheckCancelTx                       TxType = "CheckCancel"
-	CheckCashTx                         TxType = "CheckCash"
-	CheckCreateTx                       TxType = "CheckCreate"
-	ClawbackTx                          TxType = "Clawback"
-	DepositPreauthTx                    TxType = "DepositPreauth"
-	DIDDeleteTx                         TxType = "DIDDelete"
-	DIDSetTx                            TxType = "DIDSet"
-	EscrowCancelTx                      TxType = "EscrowCancel"
-	EscrowCreateTx                      TxType = "EscrowCreate"
-	EscrowFinishTx                      TxType = "EscrowFinish"
+	AccountSetTx       TxType = "AccountSet"
+	AccountDeleteTx    TxType = "AccountDelete"
+	AMMBidTx           TxType = "AMMBid"
+	AMMCreateTx        TxType = "AMMCreate"
+	AMMDeleteTx        TxType = "AMMDelete"
+	AMMDepositTx       TxType = "AMMDeposit"
+	AMMVoteTx          TxType = "AMMVote"
+	AMMWithdrawTx      TxType = "AMMWithdraw"
+	CheckCancelTx      TxType = "CheckCancel"
+	CheckCashTx        TxType = "CheckCash"
+	CheckCreateTx      TxType = "CheckCreate"
+	ClawbackTx         TxType = "Clawback"
+	DepositPreauthTx   TxType = "DepositPreauth"
+	DIDDeleteTx        TxType = "DIDDelete"
+	DIDSetTx           TxType = "DIDSet"
+	EscrowCancelTx     TxType = "EscrowCancel"
+	EscrowCreateTx     TxType = "EscrowCreate"
+	EscrowFinishTx     TxType = "EscrowFinish"
+	MPTokenAuthorizeTx TxType = "MPTokenAuthorize"
+	//nolint:gosec
```

Comment:
Why do we need this here?

Reply:
For some reason unknown to me, the gosec linter treats the variable name as a secret. It's probably because it contains the word "Token."